corehq/apps/userreports/pillow_utils.py

```diff
@@ -9,6 +10,19 @@
 from pillowtop.logger import pillow_logging
 
 
+def _is_datasource_active(adapter):
+    """
+    Tries to fetch a fresh copy of datasource from couchdb to know whether it is deactivated.
+    If it does not exist then it assumed to be deactivated
+    """
+    try:
+        config_id = adapter.config._id
+        config = adapter.config.get(config_id)
+    except ResourceNotFound:
+        return False
+    return config.is_deactivated
```

Comment:
Should this be negated?
```suggestion
    return not config.is_deactivated
```